core/src/main/java/jenkins/security/NonSerializableSecurityContext.java

```diff
@@ -40,6 +41,7 @@
  * @see hudson.security.HttpSessionContextIntegrationFilter2
  * @since 1.509
  */
+@SuppressFBWarnings(value = "SE_TRANSIENT_FIELD_NOT_RESTORED", justification = "It is intended to be not serialized and only have defaults in case of deserialization")
```

Comment:
```suggestion
@SuppressFBWarnings(value = "SE_TRANSIENT_FIELD_NOT_RESTORED", justification = "It is not intended to be serialized. Default values will be used in case of deserialization")
```